packages/web-components/@groupby/elements-sayt/stories/index.ts

```diff
@@ -296,6 +308,17 @@ storiesOf('Components|SAYT', module)
               5. Observe that the autocomplete selection changes.
 
 
+          ### The SAYT component paired with a searchbox will update the search term when an autocomplete term is selected by keydown.
+            * Pressing enter on an autocomplete term will emit an \`${UPDATE_SEARCH_TERM}\` event.
+              * This event will contain the autocomplete term with a flag to not trigger a new search as the payload.
+              * The search-box component will subsequently trigger a search.
```

Comment:
I feel like this sentence needs an "as normal" or something at the end. I say that because it sounds to some degree as if it contradicts the line above it. Why would it trigger a search if we specified that it shouldn't trigger a search?